### PR TITLE
For EmberTimes No. 73: EmberConf CfP + Ember Hooks

### PR DESCRIPTION
**Pull request by jessica-jordan**

### CODE REVIEW

File: source/blog/2018-11-16-the-ember-times-issue-73.md [hunk 1]
```diff
@@ -1,24 +1,45 @@
...
 
-<SOME-INTRO-HERE-TO-KEEP-THEM-SUBSCRIBERS-READING>
+Last call for your **talk submissions** to **EmberConf 2019** 📣!
 
 ---
 
-## [SECTION TITLE](#section-url)
+## [Your Last Chance to Become a Speaker at EmberConf 2019 🎤](https://emberconf.com/become-a-speaker.html)
+
+Have you never tried public speaking before, but you're curious to share your thoughts on something interesting you learned recently? Are you already an experienced meetup speaker and looking for the next stage? Have you spoken at conferences before and want to try something new?
```
Comment (amyrlam) [inline source/blog/2018-11-16-the-ember-times-issue-73.md]:
```suggestion
Never tried public speaking before, but you're curious to share your thoughts on something interesting you learned recently? Are you already an experienced meetup speaker and looking for the next stage? Have you spoken at conferences before and want to try something new?
```

File: source/blog/2018-11-16-the-ember-times-issue-73.md [hunk 2]
```diff
@@ -1,24 +1,45 @@
...
+  <a class="ember-button" href="https://emberconf.com/become-a-speaker.html">Ok, let me check out the CfP</a>
+</div>
+
 
 
 ---
 
-## [SECTION TITLE](#section-url)
+<!--alex ignore hooks-->
+## [Embereact Hooks ⚛️ ](https://github.com/lifeart/hooked-components)
```
Comment (amyrlam) [inline source/blog/2018-11-16-the-ember-times-issue-73.md]:
```suggestion
## [Embereact Hooks ⚛️](https://github.com/lifeart/hooked-components)
```

Comment (amyrlam) [inline source/blog/2018-11-16-the-ember-times-issue-73.md]:
```suggestion
## [Embereact Hooks ⚛️](https://github.com/lifeart/hooked-components)
```

File: source/blog/2018-11-16-the-ember-times-issue-73.md [hunk 3]
```diff
@@ -1,24 +1,45 @@
...
+author: Jessica Jordan, the crowd
 tags: Recent Posts, Newsletter, Ember.js Times, Ember Times, 2018
 alias : "blog/2018/11/16-the-ember-times-issue-73.html"
 responsive: true
 ---
 
 <SAYING-HELLO-IN-YOUR-FAVORITE-LANGUAGE> Emberistas! 🐹
 
-<SOME-INTRO-HERE-TO-KEEP-THEM-SUBSCRIBERS-READING>
+Last call for your **talk submissions** to **EmberConf 2019** 📣!
```
Comment (amyrlam) [inline source/blog/2018-11-16-the-ember-times-issue-73.md]:
```suggestion
Last call for your **talk submissions** to **EmberConf 2019** 📣! Try out React Hooks in Ember 👯‍♀️, ...
```

**Comment by jessica-jordan** [comment]
@amyrlam Thank you for all the suggestions, updated :sparkles:

**Pull request merged**